<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWindow</class>
 <widget class="QMainWindow" name="UsersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/Logo.png</normaloff>:/images/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(35, 35, 35);</string>
   </property>
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-2</y>
      <width>804</width>
      <height>484</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/BlankBackground.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/CloseButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="learningGroupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Learn</string>
    </property>
    <widget class="QLabel" name="learningStatusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera off</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="learningImageLabel2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveImageButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="learningImageLabel1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="startCameraButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>Start Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startSearchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>Start Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>124</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="infoGroupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>400</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="deleteImageButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>270</width>
       <height>140</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="userImageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imageCountLabel">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>127</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10/10</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="prevImageButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextImageButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="StaticScrollArea" name="scrollPane">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>270</width>
      <height>330</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(173,173,173);</string>
    </property>
    <property name="text">
     <string>Users</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StaticScrollArea</class>
   <extends>QFrame</extends>
   <header>widgets/staticscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
